### ISSUE TITLE
(Suggestion / Request) Sliders

### ISSUE THREAD

**Issue by urtaur** [baconpaul/airwin2rack#166]
It would be very difficult to create a variation in the sliders tipology? 
I am thinking in an option in the "Settings" menu to change the circular sliders to horizontal ones with the numbers/parameters below. 
I know it is possible to load the individual plugins without your consolidated plugin, but, me, as a Reaper user, I can not type numerical values that way. And your menus and gain in and out are very convenient.

By the way... thanks for the plugin!

**Comment by baconpaul** [comment]
That's probably an afternoon's work, yeah. Knobs are just sliders that paint wrong. But curious if the desire for that is anything other than aesthetic?

**Comment by urtaur** [comment]
It is not only aesthetic. I think it is easier for some people to dial them (as it is my case). Perhaps i am so used to Reaper and some JSFX effects in it, but, in general (except very big circular sliders), i prefer horizontal sliders to dial and to visualize the values.

Yes... I enjoy your plugin!

**Comment by baconpaul** [comment]
yeah the throw is basically the same in either case just vertical vs horizontal mouse movement. but got it. 

anyway not super high priority of course. but i'll leave the issue open in case its a rainy day and I feel like some UI code!

**Comment by baconpaul** [comment]
oh you know also if you press shift while dragging the knobs have a wider sensitibity? And you a type into the params in the UI if you want an exact value.

**Comment by urtaur** [comment]
"oh you know also if you press shift while dragging the knobs have a wider sensitibity? And you a type into the params in the UI if you want an exact value."

Yes, I know it. 
I hope a raini day will come! XD

I understand it couldn´t be an high priority task. Thank you very much. Great job!
